packages/nxengine/build.sh

```diff
@@ -0,0 +1,39 @@
+TERMUX_PKG_NAME=NXEngine
+TERMUX_PKG_HOMEPAGE=https://nxengine.sourceforge.net
+TERMUX_PKG_DESCRIPTION="Open-source rewrite engine of the Cave Story for Dingux and MotoMAGX"
+TERMUX_PKG_LICENSE="GPL-3.0"
+TERMUX_PKG_MAINTAINER="@Yonle"
+TERMUX_PKG_VERSION=1.0.0.4-Rev4
+TERMUX_PKG_SKIP_SRC_EXTRACT=true
+TERMUX_PKG_DEPENDS="sdl, sdl-ttf, pulseaudio"
+TERMUX_PKG_BUILD_IN_SRC=true
+
+termux_step_get_source() {
+	git clone https://github.com/EXL/NXEngine $TERMUX_PKG_SRCDIR
```

Comment:
This is not correct way of fetching sources from git. At least this doesn't meet our requirements.

build-package.sh defines a way to make git clone with TERMUX_PKG_SRCURL.

Don't stick to latest commit or something else which is not controlled through metadata.

Reply:
Well i don't think that `build-package.sh` also can do git clone.

In release of NXengine, Buolding from that release is broken and fail to build anyway